lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc

```diff
@@ -98,11 +98,14 @@ function dosomething_signup_form_submit($form, &$form_state) {
   }
 
   if (variable_get('rogue_collection', FALSE)) {
+    $form_state['values']['source'] = 'phoenix-web';
```

Comment:
Does this normally have more specific source information? Like `fastweb` or `node/1141`? If so, we should probably send it along so we make sure we don't lose any data.